### PR TITLE
Create a guide for showing how to connect algob project settings with a webapp

### PR DESCRIPTION
**Pull request by ratik21**

### CODE REVIEW

File: docs/guide/algob-web.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Webapp:
+├── assets
+│   ├── TEAL files
+│   ├── PyTEAL files
+├── scripts (algob project scripts)
+│   ├── deploy.ts
+│   ├── run.ts
+├── src (react frontend code)
+│   ├── app.js
+│   ├── index.jjs
```
Comment (amityadav0) [inline docs/guide/algob-web.md:28]:
```suggestion
│   ├── index.js
```

File: docs/guide/algob-web.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+│   ├── .mocharc.json (optional)
+├── algob.config.js
+├── package.json (common for algob & react-app)
+```
+
+Contracts are present in `/assets`, scripts are present in `/scripts` folder, and dapp's frontend code is available in `/src` folder.
+
+## Checkpoints
+
+In an `algob` project we have persist [checkpoints](./algob-web-checkpoints.md). This is helpful to store deployment information (transaction IDs, asset indexes, etc.) for later use (in future scripts).
```
Comment (amityadav0) [inline docs/guide/algob-web.md:40]:
```suggestion
In an `algob` project we persist [checkpoints](./algob-web-checkpoints.md). This is helpful to store deployment information (transaction IDs, asset indexes, etc.) for later use (in future scripts).
```

File: docs/guide/algob-web.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
++ The checkpoint files are saved in `artifacts/scripts/` in a human-readable YAML format. Contains:
+    + Deployed informaton (assetID, appID, creator, txID, etc.)
+    + Logic Signatures (Delegated/signed OR unsigned)
++ Cached information (eg. compiled smart contracts) are stored in `artifacts/cache` (contract filename, compiled hash, TEAL code, etc.)
+
+Since, `@algo-builder/web` API can't load data from files in your webapp, user either needs to:
++ copy these checkpoints from `/artifacts` to your webapp (inside `/src/`), so it can be bundled by webpack and deployed.
++ OR, you can override your react-app config and remove `removeModuleScopePlugin` (this allows imports outside `/src` folder in react-app)
+
+After you're able to import checkpoint files (in `/artifacts/*.yaml`), you can simply add a yaml loader to load files & read data in your react source code. For eg. (reading a logic sig from checkpoints data and getting the contract's address).
```
Comment (amityadav0) [inline docs/guide/algob-web.md:50]: could you please add code snippet for yaml loading ?

Comment (ratik21) [reply docs/guide/algob-web.md:50]: It's a simple loading (import * from "artifacts/<name>.yml"). I can add a code snippet for config-override

**Review by amityadav0 — APPROVED** [review]

**Pull request merged**